.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## PR Checklist
+
+For a smooth review process, please run through this checklist before submitting a PR.
```

Comment:
We can make room for a description to make it easier to fill in. Also we can use comments to hide parts of the template after the PR is opened.

```suggestion
## Description

<!-- Which issue/s does this PR close? Is there any more context you can give the reviewer? -->

## PR Checklist

<!-- For a smooth review process, please run through this checklist before submitting a PR. -->
```